migrations/20210512203600-add-shopper-references.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    /**
+     * Add altering commands here.
+     *
+     * Example:
+     * await queryInterface.createTable('users', { id: Sequelize.INTEGER });
+     */
+    await queryInterface.addConstraint('Shoppers', {
+      fields: ['UserId'],
+      type: 'FOREIGN KEY',
+      references: {
+        table: 'Users',
+        field: 'id'
+      }
+    });
+    await queryInterface.addConstraint('Shoppers', {
+      fields: ['UserId'],
+      type: 'UNIQUE'
+    });
+  },
```

Comment:
Here I add the FOREIGN KEY constraint _and_ the UNIQUE constraint that makes User-Shopper a ONE-TO-ONE relationship.